src/s2/s2loop.h

```diff
@@ -640,12 +637,8 @@ class S2Loop final : public S2Region {
   // We store the vertices in an array rather than a vector because we don't
   // need any STL methods, and computing the number of vertices using size()
   // would be relatively expensive (due to division by sizeof(S2Point) == 24).
-  // When DecodeWithinScope is used to initialize the loop, we do not
-  // take ownership of the memory for vertices_, and the owns_vertices_ field
-  // is used to prevent deallocation and overwriting.
   int num_vertices_ = 0;
-  S2Point* vertices_ = nullptr;
-  bool owns_vertices_ = false;
```

Comment:
I don't see a good reason why we can't, but let's do it as a separate PR.

Reply:
Of course, I just interested

Reply:
> Btw I think feature with reading vertices from coded memory wasn't bad.

It was deleted because it was unused and warty. It was a hack used in at most 3 places at its peak popularity. I think the current way is to use encoded `S2Shape`s.